portfolio/src/main/webapp/index.html

```diff
@@ -131,9 +137,50 @@ <h3 class= "iHead" >Stuff I Like</h3>
 	    
 	  </div>
 	</div>
+
+
+	<!-- Overall container for comments form + old comments -->
+	<div id = "comments">
+	  
+	  <form action="/data" method="POST">
```

Comment:
The text boxes aren't lined up, might be good to change/add some padding or something so they're lined up and the text is inline?